Blockzilla/UIComponents/UIConstants.swift

```diff
@@ -335,6 +335,11 @@ struct UIConstants {
         static let defaultBrowserOnboardingViewTopButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.TopButtonTitle.V2", value: "Set as Default Browser", comment: "Text for a label that indicates the title of the top button from the default browser onboarding screen version 2.")
         static let defaultBrowserOnboardingViewBottomButtonTitleV2 = NSLocalizedString("Onboarding.DefaultBrowser.BottomButtonTitle.V2", value: "Skip", comment: "Text for a label that indicates the title of the bottom button from the default browser onboarding screen version 2.")
         static let tooltipBodyTextForTrashIconV2 = NSLocalizedString("TooltipBodyText.TrashIcon.V2", value: "Tap here to trash it all — history, cookies, everything — and start fresh on a new tab.", comment: "This is the body text that is displayed for the Trash icon tooltip")
+        static let titleShowMeHowOnboardingV2 = NSLocalizedString("ShowMeHowOnboarding.Title.V2", value: "Add a Focus Widget", comment: "This is the title text that is displayed in the Show Me How Onboarding Screen")
```

Comment:
Once again, you can't have Focus hard-coded in strings.

At this point, I strongly suggest you create actiive safeguards against this (linter).

Reply:
Yes, I'm thinking of something. Bear with us. Thanks for spotting them.